flake.nix

```diff
@@ -0,0 +1,62 @@
+{
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+
+    naersk.url = "github:nix-community/naersk";
+
+    nixpkgs-mozilla = {
+      url = "github:mozilla/nixpkgs-mozilla";
+      flake = false;
+    };
+
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+
+  };
+
+  outputs = { self, flake-utils, nixpkgs, nixpkgs-mozilla, naersk }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = (import nixpkgs) {
+          inherit system;
+
+          overlays = [
+            (import nixpkgs-mozilla)
+          ];
+        };
+
+        toolchain = (pkgs.rustChannelOf {
+          rustToolchain = ./rust-toolchain;
+          sha256 = "sha256-rLP8+fTxnPHoR96ZJiCa/5Ans1OojI7MLsmSqR2ip8o=";
+        }).rust;
+
+        naersk' = pkgs.callPackage naersk {
+          cargo = toolchain;
+          rustc = toolchain;
+        };
+
+        fastnp = naersk'.buildPackage {
+          name = "fastn";
+          version = "0.3.0";
+          src = ./.;
+
+          nativeBuildInputs = with pkgs; [ pkg-config openssl.dev ];
```

Comment:
I found [this PR](https://github.com/NixOS/nixpkgs/pull/96996/files) which suggests that adding `xcbuild` should fix this.
I do not have a Mac to test this, can you try adding `xcbuild` to the `nativeBuildInputs` here?

Use `nix run .` instead of the `nix run github:fastn-stack/fastn` to test it locally without pushing it to github.

Reply:
@siddhantk232 This did not help:

```nix
        devShell = pkgs.mkShell {
          nativeBuildInputs = [ toolchain pkgs.pkg-config pkgs.openssl.dev pkgs.xcbuild ];
        };
```

Same error.